eq-publisher/src/eq_schema/builders/routing2/translateBinaryExpression.test.js

```diff
@@ -174,5 +175,28 @@ describe("Should build a runner representation of a binary expression", () => {
         });
       });
     });
+
+    it("can translate unanswered question routing from Author to Runner for all types", () => {
```

Comment:
Should it be "for all **numeric** types"?

Reply:
Yeah, that probably makes the most sense. Cheers!